### PR TITLE
Roadmap navigation

### PR DESCRIPTION
**Pull request by jcastle**
Updated navigation of site to accommodate finding the Roadmap.  Also fixed spacing issues with the page.

### CODE REVIEW

File: src/app/components/search-results/search-results.template.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <div class="search-results-content">
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
   <div class="search-results-banner">
     <h1>Search Results</h1>
   </div>
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
   <div class="search-results-breadcrumb usa-grid">
     <ul class="usa-unstyled-list">
       <li><a routerLink="/">Home</a></li>
       <li>Search Results</li>
     </ul>
   </div>
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html [hunk 4]
```diff
@@ -68,8 +72,10 @@ <h5>Usage Type</h5>
         </fieldset>
       </form>
     </div> -->
+
     <div class="search-results-section">
       <repo-list [queryValue]="queryValue" [results]="filteredResults" [pageSize]="pageSize"></repo-list>
     </div>
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html [hunk 5]
```diff
@@ -68,8 +72,10 @@ <h5>Usage Type</h5>
         </fieldset>
       </form>
     </div> -->
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html [hunk 6]
```diff
@@ -30,6 +33,7 @@ <h3 *ngIf="total" class="repos-count">
       </p>
     </form>
   </div>
+
```
Comment (froi) [inline src/app/components/search-results/search-results.template.html]: Please remove whitespace.

File: src/app/components/search-results/search-results.template.html (hunk 1 above)
Comment (jcastle) [reply src/app/components/search-results/search-results.template.html]: What!!!

**Pull request merged**